spec/SPEC.md

```diff
@@ -0,0 +1,301 @@
+responsible for identifying any or all such patent rights.
+
+Any trade name used in this document is information given for the convenience
+of users and does not constitute an endorsement.
+
+This document was prepared by the OmniBOR Community.
+
+Known patent licensing exclusions are available in the specification
+repository's `NOTICES.md` file.
+
+Any feedback or questions on this document should be directed to the
+specification's repository, located at <https://github.com/omnibor/spec>.
+
+THESE MATERIALS ARE PROVIDED "AS IS." The Contributors and Licensees expressly
+disclaim any warranties (express, implied, or otherwise), including implied
+warranties of merchantability, non-infringement, fitness for a particular
+purpose, or title, related to the materials. The entire risk as to
+implementing or otherwise using the materials is assumed by the implementer and
+user. IN NO EVENT WILL THE CONTRIBUTORS OR LICENSEES BE LIABLE TO ANY OTHER
+PARTY FOR LOST PROFITS OR ANY FORM OF INDIRECT, SPECIAL, INCIDENTAL, OR
+CONSEQUENTIAL DAMAGES OF ANY CHARACTER FROM ANY CAUSES OF ACTION OF ANY KIND
+WITH RESPECT TO THIS DELIVERABLE OR ITS GOVERNING AGREEMENT, WHETHER BASED ON
+BREACH OF CONTRACT, TORT (INCLUDING NEGLIGENCE), OR OTHERWISE, AND WHETHER OR
+NOT THE OTHER MEMBER HAS BEEN ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+## Introduction
+
+Software supply chains face many challenges: security and compliance chief
+among them. Often, projects are hamstrung by the inability to easily and
+reliably capture a complete, concise, verifiable accounting of exactly
+__what__ inputs were built into software. Without this information,
+identifying vulnerable software to patch or replace is difficult. While
+Software Bills of Material (SBOMs) help identify third-party components,
+they do not go far enough to precisely identify the exact inputs necessary
+for vulnerability management.
+
+The OmniBOR standard defines three concepts, which together enable the
+consistent, reproducible, and embeddable encoding of the exact inputs used to
+build a software artifact: Artifact Identifiers, Input Manifests, and Artifact
+Dependency Graphs.
+
+An Artifact Identifier is a content-based identifier of a single input (for
+example, a single file) used to build a software artifact. Identifiers are
+reproducible, meaning two individuals will always derive the same identifier
+for the same input. With these identifiers, we can consistently and precisely
+identify any software artifact or its input, for use in forensics, accounting,
+and vulnerability management.
+
+Next, an Input Manifest lists the Artifact Identifier of every input used to
+produce an artifact. For example, if an executable is compiled by linking
+together a collection of object files, the Artifact Identifier of every object
+file would be listed in the Input Manifest for the executable. Input manifests
+can be embedded directly into executable files, or can be provided in a
```

Comment:
> Input manifests can be embedded directly

I think what we want to say is that Input manifest _references_ can be embedded directly, not the entire manifest itself.